base/src/main/proto/spine/options.proto

```diff
@@ -442,6 +442,20 @@ extend google.protobuf.MessageOptions {
     // Otherwise, please use fully qualified names.
     //
     string rejections = 73940;
+
+    // General purpose options
+    //-------------------------
+
+    // Specifies the name of a Java interface to be implemented by this message.
+    //
+    // The interface is generated into the Protobuf generation destination directory. The interface
+    // itself is a marker interface derived from `com.google.protobuf.Message`. Unless the interface
+    // is specified by it's FQN, it will have the same Java package as the message.
+    //
+    // To specify a marker interface for every message in a `.proto` file, use `(every_is)` file
+    // option. If both `(is)` and `(every_is)` options are found, `(is)` wins.
+    //
+    string is = 73941;
```

Comment:
Let's move this to `API Annotations` taking 73915. This option is much more related to API that we declare by marking the message with a marker interface, than to some ephemeral general purpose options.

Then the changes to the _existing_ tag numbers made below can be reverted.